ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -169,34 +196,101 @@ tests:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
 - as: aws-ipi-private-f28
-  cron: 36 12 9 * *
+  cron: 40 12 6 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
-- as: aws-ipi-proxy-sts-f14
-  cron: 4 18 10,25 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
+  cron: 24 4 8 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: aws-ipi-proxy-sts-f28
+  cron: 27 11 15 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-registry-s3-cloudfront-f28
+  cron: 5 14 9 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
+- as: aws-ipi-rm-root-secret-f28
```

Comment:
Can you help check if this install workflow need to be regularly covered in upgrade ci?

Reply:
hi @yunjiang29 I searched the whole repo, and see the oldest version with this profile is openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.12.yaml which owner is you. Could you please help confirm can we add this profile to 4.11 stable upgrade to 4.12 nightly?

Reply:
I removed this profiles in last commit, if we can run it in 4.11 to 4.12 upgrade path, I will add it back.

Reply:
_rm-root-secret_ configuration is only applicable for z release upgrade